### ISSUE TITLE
【EESS】New submissions for Mon, 24 Jul 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#453]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Prediction of sunflower leaf area at vegetative stage by image analysis  and application to the estimation of water stress response parameters in  post-registration varieties
 - **Authors:** Pierre Casadebaig (INRAE), Nicolas Blanchet (INRAE), Nicolas Bernard Langlade (INRAE)
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2307.11110
 - **Pdf link:** https://arxiv.org/pdf/2307.11110
 - **Abstract**
 The automatic measurement of developmental and physiological responses of sunflowers to water stress represents an applied challenge for a better knowledge of the varieties available to growers, but also a fundamental one for identifying the biological, genetic and molecular bases of plant response to their environment.On INRAE Toulouse's Heliaphen high-throughput phenotyping platform, we set up two experiments, each with 8 varieties (2*96 plants), and acquired images of plants subjected or not to water stress, using a light barrier on a daily basis. At the same time, we manually measured the leaf surfaces of these plants every other day for the duration of the stress, which lasted around ten days. The images were analyzed to extract morphological characteristics of the segmented plants and different models were evaluated to estimate total plant leaf areas using these data.A linear model with a posteriori smoothing was used to estimate total leaf area with a relative squared error of 11% and an efficiency of 93%. Leaf areas estimated conventionally or with the developed model were used to calculate the leaf expansion and transpiration responses (LER and TR) used in the SUNFLO crop model for 8 sunflower varieties studied. Correlation coefficients of 0.61 and 0.81 for LER and TR respectively validate the use of image-based leaf area estimation. However, the estimated values for LER are lower than for the manual method on Heliaphen, but closer overall to the manual method on greenhouse-grown plants, potentially suggesting an overestimation of stress sensitivity.It can be concluded that the LE and TR parameter estimates can be used for simulations. The low cost of this method (compared with manual measurements), the possibility of parallelizing and repeating measurements on the Heliaphen platform, and of benefiting from the Heliaphen platform's data management, are major improvements for valorizing the SUNFLO model and characterizing the drought sensitivity of cultivated varieties.
### Dehazing Ultrasound using Diffusion Models
 - **Authors:** Tristan S.W. Stevens, Faik C. Meral, Jason Yu, Iason Z. Apostolakis, Jean-Luc Robert, Ruud J.G. van Sloun
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.11204
 - **Pdf link:** https://arxiv.org/pdf/2307.11204
 - **Abstract**
 Echocardiography has been a prominent tool for the diagnosis of cardiac disease. However, these diagnoses can be heavily impeded by poor image quality. Acoustic clutter emerges due to multipath reflections imposed by layers of skin, subcutaneous fat, and intercostal muscle between the transducer and heart. As a result, haze and other noise artifacts pose a real challenge to cardiac ultrasound imaging. In many cases, especially with difficult-to-image patients such as patients with obesity, a diagnosis from B-Mode ultrasound imaging is effectively rendered unusable, forcing sonographers to resort to contrast-enhanced ultrasound examinations or refer patients to other imaging modalities. Tissue harmonic imaging has been a popular approach to combat haze, but in severe cases is still heavily impacted by haze. Alternatively, denoising algorithms are typically unable to remove highly structured and correlated noise, such as haze. It remains a challenge to accurately describe the statistical properties of structured haze, and develop an inference method to subsequently remove it. Diffusion models have emerged as powerful generative models and have shown their effectiveness in a variety of inverse problems. In this work, we present a joint posterior sampling framework that combines two separate diffusion models to model the distribution of both clean ultrasound and haze in an unsupervised manner. Furthermore, we demonstrate techniques for effectively training diffusion models on radio-frequency ultrasound data and highlight the advantages over image data. Experiments on both \emph{in-vitro} and \emph{in-vivo} cardiac datasets show that the proposed dehazing method effectively removes haze while preserving signals from weakly reflected tissue.
### BLISS: Interplanetary Exploration with Swarms of Low-Cost Spacecraft
 - **Authors:** Alexander N. Alvara, Lydia Lee, Emmanuel Sin, Nathan Lambert, Andrew J. Westphal, Kristofer S.J. Pister
 - **Subjects:** Systems and Control (eess.SY); Image and Video Processing (eess.IV); Space Physics (physics.space-ph)
 - **Arxiv link:** https://arxiv.org/abs/2307.11226
 - **Pdf link:** https://arxiv.org/pdf/2307.11226
 - **Abstract**
 Leveraging advancements in micro-scale technology, we propose a fleet of autonomous, low-cost, small solar sails for interplanetary exploration. The Berkeley Low-cost Interplanetary Solar Sail (BLISS) project aims to utilize small-scale technologies to create a fleet of tiny interplanetary femto-spacecraft for rapid, low-cost exploration of the inner solar system. This paper describes the hardware required to build a nearly 10 g spacecraft using a 1 m2 solar sail steered by micro-electromechanical systems (MEMS) inchworm actuators. The trajectory control to a NEO, here 101955 Bennu, is detailed along with the low-level actuation control of the solar sail and the specifications of proposed onboard communication and computation. Two other applications are also shortly considered: sample return from dozens of Jupiter-family comets and interstellar comet rendezvous and imaging. The paper concludes by discussing the fundamental scaling limits and future directions for steerable autonomous miniature solar sails with onboard custom computers and sensors.
### An Improved Nonlinear Stability Assessment Methodology For Type-4 Wind  Turbines via Time Reversal Trajectory
 - **Authors:** Sujay Ghosh, Mohammad Kazem Bakhshizadeh, Guangya Yang, Łukasz Kocewiak
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2307.11445
 - **Pdf link:** https://arxiv.org/pdf/2307.11445
 - **Abstract**
 As the integration of renewable energy generation increases and as conventional generation is phased out, there is a gradual decline in the grid's strength and resilience at the connection point of wind turbines (WTs). Previous studies have shown that traditional grid-following controlled converters exhibit deteriorating dynamic characteristics and may result in an unstable system when connected to a weak grid. Due to the limitations of linear analysis, transient stability investigations are necessary. However, existing methods, such as standalone time-domain simulations or analytical Lyapunov stability criteria, have drawbacks, including computational intensity or excessive conservatism. Our prior research proposed an innovative approach to estimate the system boundary - a time-limited region of attraction (TLRoA), using a hybrid linearised Lyapunov function-based method and the time-reversal technique to compensate for the known limitations. However, in that work, the accuracy of the estimated TLRoA was not investigated, i.e. the TLRoA was not compared against a forward simulated region of attraction, and the sensitivity of the system parameters on the TLRoA was not explored. Moreover, the framework did not consider nonlinear control elements such as PLL saturation. In this paper, we not only build upon our previous work and propose directions that address these gaps but also enhance its effectiveness by introducing optimal sampling to improve further the speed of estimating the TLRoA. Furthermore, the stability boundary is verified using time-domain simulation studies in PSCAD.
### Data-Driven Cooperative Adaptive Cruise Control for Unknown Nonlinear  Vehicle Platoons
 - **Authors:** Jianglin Lan
 - **Subjects:** Systems and Control (eess.SY); Robotics (cs.RO); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2307.11505
 - **Pdf link:** https://arxiv.org/pdf/2307.11505
 - **Abstract**
 This paper studies cooperative adaptive cruise control (CACC) for vehicle platoons with consideration of the unknown nonlinear vehicle dynamics that are normally ignored in the literature. A unified data-driven CACC design is proposed for platoons of pure automated vehicles (AVs) or of mixed AVs and human-driven vehicles (HVs). The CACC leverages online-collected sufficient data samples of vehicle accelerations, spacing and relative velocities. The data-driven control design is formulated as a semidefinite program (SDP) that can be solved efficiently using off-the-shelf solvers. The efficacy and advantage of the proposed CACC are demonstrated through a comparison with the classic adaptive cruise control (ACC) method on a platoon of pure AVs and a mixed platoon under a representative aggressive driving profile.
### Amplitude-Based Sequential Optimization of Energy Harvesting with  Reconfigurable Intelligent Surfaces
 - **Authors:** Morteza Tavana, Meysam Masoudi, Emil Björnson
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.11523
 - **Pdf link:** https://arxiv.org/pdf/2307.11523
 - **Abstract**
 Reconfigurable Intelligent Surfaces (RISs) have gained immense popularity in recent years because of their ability to improve wireless coverage and their flexibility to adapt to the changes in a wireless environment. These advantages are due to RISs' ability to control and manipulate radio frequency (RF) wave propagation. RISs may be deployed in inaccessible locations where it is difficult or expensive to connect to the power grid. Energy harvesting can enable the RIS to self-sustain its operations without relying on external power sources. In this paper, we consider the problem of energy harvesting for RISs in the absence of coordination with the ambient RF source. We consider both direct and indirect energy harvesting scenarios and show that the same mathematical model applies to them. We propose a sequential phase-alignment algorithm that maximizes the received power based on only power measurements. We prove the convergence of the proposed algorithm to the optimal value under specific circumstances. Our simulation results show that the proposed algorithm converges to the optimal solution in a few iterations and outperforms the random phase update method in terms of the number of required measurements.
### Data-based system representations from irregularly measured data
 - **Authors:** Mohammad Alsalti, Ivan Markovsky, Victor G. Lopez, Matthias A. Müller
 - **Subjects:** Systems and Control (eess.SY); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2307.11589
 - **Pdf link:** https://arxiv.org/pdf/2307.11589
 - **Abstract**
 Non-parametric representations of dynamical systems based on the image of a Hankel matrix of data are extensively used for data-driven control. However, if samples of data are missing, obtaining such representations becomes a difficult task. By exploiting the kernel structure of Hankel matrices of irregularly measured data generated by a linear time-invariant system, we provide computational methods for which any complete finite-length behavior of the system can be obtained. For the special case of periodically missing outputs, we provide conditions on the input such that the former result is guaranteed. We illustrate with an example how the resulting representation provides a more computationally efficient method for low-rank matrix completion when compared to an alternative method.
### Deep Learning Hyperspectral Pansharpening on large scale PRISMA dataset
 - **Authors:** Simone Zini, Mirko Paolo Barbato, Flavio Piccoli, Paolo Napoletano
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2307.11666
 - **Pdf link:** https://arxiv.org/pdf/2307.11666
 - **Abstract**
 In this work, we assess several deep learning strategies for hyperspectral pansharpening. First, we present a new dataset with a greater extent than any other in the state of the art. This dataset, collected using the ASI PRISMA satellite, covers about 262200 km2, and its heterogeneity is granted by randomly sampling the Earth's soil. Second, we adapted several state of the art approaches based on deep learning to fit PRISMA hyperspectral data and then assessed, quantitatively and qualitatively, the performance in this new scenario. The investigation has included two settings: Reduced Resolution (RR) to evaluate the techniques in a supervised environment and Full Resolution (FR) for a real-world evaluation. The main purpose is the evaluation of the reconstruction fidelity of the considered methods. In both scenarios, for the sake of completeness, we also included machine-learning-free approaches. From this extensive analysis has emerged that data-driven neural network methods outperform machine-learning-free approaches and adapt better to the task of hyperspectral pansharpening, both in RR and FR protocols.
